clients/fidesui/src/components/floating-menu/FloatingMenu.tsx

```diff
@@ -0,0 +1,15 @@
+import { AntMenu } from "fidesui";
+import { ComponentProps } from "react";
+
+import styles from "./FloatingMenu.module.scss";
+
+type AntMenuProps = ComponentProps<typeof AntMenu>;
+
+export const FloatingMenu = ({ className, ...props }: AntMenuProps) => (
```

Comment:
Yes, I agree we shouldn't have one offs in fides-ui. In my mind, I thought given the specific design specs for this floating menu, special padding, rounded corners, the shadow and the possibility of reuse in future interactive screens like this one, that it was enough to add it to our design system. But I'll ask Jack if he thinks this is more of a one-off or something we'll be reusing in upcoming designs.

Reply:
Talked with Jack. He is on board with having this as part of the design system since he's planning on reusing this floating menu for all upcoming React Flow screens. I'll leave it as part of fidesui then.